e2etest/zt_managed_disks_test.go

```diff
@@ -20,5 +20,56 @@
 
 package e2etest
 
+import (
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"testing"
+)
+
 // Purpose: Tests for the special cases that relate to moving managed disks (default local VHD to page blob; special handling for
 //     md- and md-impex URLs.
+
+func TestManagedDisks_NoOAuthRequired(t *testing.T) {
```

Comment:
Can we have a negative test case where access is revoked and validate transfer attempt after that fails.

Reply:
This feels like it leaves the boundary of testing AzCopy and enters the boundary of testing the service. I don't see the value in adding a test like this.